src/Loop/Driver.php

```diff
@@ -15,7 +15,14 @@
     private $registry = [];
 
     /**
-     * Start the event loop.
+     * Run the event loop.
+     *
+     * One iteration of the loop is called one "tick". A tick covers the following steps:
+     *
+     *  1. Activate watchers created / enabled in the last tick / before `run`.
```

Comment:
With \`run\` you are referencing the method? Please suffix `()` then to make this more obvious.